h/static/scripts/annotator/guest.coffee

```diff
@@ -1,7 +1,20 @@
-Promise = require('es6-promise').Promise
+raf = require('raf')
+Promise = global.Promise ? require('es6-promise').Promise
```

Comment:
It's actually easier than that. `require('es6-promise')` runs the polyfill automatically but exports the shim. The solution is just to `require('es6-polyfill')` but ignore its exports rather than using them.